### PR TITLE
Add bulk element mappings for triangles.

### PR DESCRIPTION
**Pull request by TomFischer**
1. [x] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.3.1)
2. [x] Tests covering your feature were added?
3. [x] Any new feature or behavior change was documented?
         Yes, I created a pdf. The question is where to put the sketches?

### CODE REVIEW

File: MeshLib/Elements/MapBulkElementPoint.h [hunk 1]
```diff
@@ -17,6 +17,18 @@
 
 namespace MeshLib
 {
+/// Maps the given lower dimensional boundary point \c wp of a line, i.e. the 1d
+/// gauss point given in local coordinates of a line, to higher dimensional
+/// point of the tri face (defined by the tri element and the face id) also in
+/// local coordinates of the tri face.
+/// \param tri the tri element
+/// \param face_id the id of the tri face the point will be mapped on
+/// \param wp the gauss point of the lower dimensional element
```
Comment (endJunction) [inline MeshLib/Elements/MapBulkElementPoint.h:26]: Minor; Would call it integration point at least.
Better would be to call it something like local coordinate of the line element. (and then to pass only the coordinates w/o the integration weight.) But this is a long story...

File: MeshLib/Elements/MapBulkElementPoint.cpp [hunk 2]
```diff
@@ -14,6 +14,25 @@
...
+        case 0:
+            return MathLib::Point3d{std::array<double, 3>{{wp[0], 0.0, 0.0}}};
+        case 1:
+            return MathLib::Point3d{
+                std::array<double, 3>{{1 - wp[0], wp[0], 0.0}}};
+        case 2:
+            return MathLib::Point3d{
+                std::array<double, 3>{{0.0, 1 - wp[0], 0.0}}};
+        default:
+            OGS_FATAL("Invalid face id '%u' for the tri.", face_id);
```
Comment (wenqing) [inline MeshLib/Elements/MapBulkElementPoint.cpp:32]:
```suggestion
            OGS_FATAL("Invalid face id '{:d}' for the tri.", face_id);
```

Comment (TomFischer) [reply MeshLib/Elements/MapBulkElementPoint.cpp:32]: Yes, you are right. Will fix it tomorrow.

**Comment by TomFischer** [comment]
I don't see the reason for the doxygen warning:
```
argument 'quad' of command @param is not found in the argument list of MeshLib::getBulkElementPoint(MeshLib::Tri const &tri, std::size_t const face_id, MathLib::WeightedPoint1D const ℘)
```
Seems to be a false positive?

**Pull request merged**